packages/appstore/src/components/wallet-cards-carousel/wallet-cards-carousel.scss

```diff
@@ -0,0 +1,111 @@
+.wallet-cards-carousel {
+    margin: -1.6rem -1.6rem -1.4rem;
+    padding: 2.4rem 0 1.6rem;
+
+    &__viewport {
+        overflow: hidden;
+    }
+    &__container {
+        height: 100%;
+        display: flex;
+        align-items: center;
+        justify-content: flex-start;
+        gap: 2.4rem;
+    }
+    &__card {
+        width: 23rem;
+        height: 29rem;
+        padding: 1.6rem;
+        border-radius: 1.6rem;
+        border: 0.1rem solid var(--general-active);
+        background-color: var(--general-main-1);
+        box-shadow: 0px 12px 16px -4px rgba(14, 14, 14, 0.08), 0px 4px 6px -2px rgba(14, 14, 14, 0.03);
```

Comment:
@sergei-deriv you can define this shadow in the constants.scss file for further use